int32 fc_id

geometry_msgs/Point[] cell_list

geometry_msgs/Point center

geometry_msgs/Point viewpoint

geometry_msgs/Point yaw
